style.css

```diff
@@ -29,6 +31,147 @@ nav ul li a {
   color: white;
 }
 
+footer ul {
+  margin-top: 120px;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+footer ul li {
+  list-style: none;
+  color: #505f79;
+}
+
+.model-tech {
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.model-tech ul {
+  display: flex;
+  list-style-type: none;
+  padding: 5px;
+  margin-top: 5px;
+}
+
+.model-tech ul li {
+  border: 1px solid black;
+  margin-right: 5px;
+  padding: 5px;
+}
+
+.model-img img {
+  width: 100%;
+}
+
+.model p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.model-btn a {
+  text-decoration: none;
+  padding: 5px;
+  background-color: #ff6b00;
+  color: white;
+  border: none;
+  display: flex;
+  align-items: center;
+  margin-right: 5px;
+}
+
+.title_container img {
+  width: 18px;
+  height: 18px;
+}
+
+.model-btn a img {
+  margin-left: 5px;
+}
+
+.model-btn {
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.popup {
+  overflow-y: scroll;
+  display: none;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 2;
+}
```

Comment:
- [ ] Great job here 💯 However, according to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). So kindly make the popup a little bit at the top You can do this by increasing `top: 92px;`, and make the image and paragraph on the same level 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/195536762-dd3aba79-bb39-4003-9cc7-f79ed9f0a7da.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/195537137-a4d60ac7-b4f6-4c2a-bece-6a2be1ab7604.png" width="500" /> |